e2e/_suites/kubernetes-autodiscover/autodiscover_test.go

```diff
@@ -158,13 +158,24 @@ func (m *podsManager) configureDockerImage(podName string) error {
 			return err
 		}
 
-		// tag the image with the proper docker tag, including platform
-		err = deploy.TagImage(
-			"docker.elastic.co/beats/"+podName+":"+downloads.GetSnapshotVersion(common.BeatVersionBase),
-			"docker.elastic.co/observability-ci/"+podName+":"+beatVersion,
-		)
-		if err != nil {
-			return err
+		if podName == "elastricsearch" {
```

Comment:
typo:
```suggestion
		if podName == "elasticsearch" {
```

Reply:
I'd define a `namespace := "beats"` variable, assigning it to `namespace = "elasticsearch"` if `podName == "elasticsearch"`. Then the below logic would not be duplicated, using the image's fully qualified name the namespace as part of the string.